README.md

```diff
@@ -125,6 +126,28 @@ method. Using it will overwrite specified properties and leave others unchanged.
 Any spans reported from this point on will have your new attributes set.  
 You can pass `{}` or `undefined` to clear your global attributes.
 
+### Redacting PII
+In certain situations, metadata collected by our instrumentation may include PII.
+We'd advise that you review 2 cases in particular:
+- any network operation, where a secret piece of information might be present in the URL (e.g. an authentication token)
```

Comment:
"Please note that we do not capture or report any data from the payload of the request (i.e. the POST body), apart from its size"